packages/suite-desktop/src-electron/modules/tor.ts

```diff
@@ -56,9 +56,45 @@ const load = async ({ mainWindow, store, interceptor }: Dependencies) => {
 
         if (shouldEnableTor === true) {
             setProxy(`socks5://${host}:${port}`);
-            await tor.start();
+            tor.torController.on(
+                'bootstrap/event',
+                (bootstrapEvent: { progress: string; summary: string }) => {
+                    if (bootstrapEvent && bootstrapEvent.summary && bootstrapEvent.progress) {
+                        logger.info(
+                            'tor',
+                            `Bootstrap - ${bootstrapEvent.progress || ''}% - ${
+                                bootstrapEvent.summary || ''
+                            }`,
+                        );
+
+                        mainWindow.webContents.send('tor/bootstrap', {
+                            type: 'progress',
+                            summary: bootstrapEvent.summary,
+                            progress: {
+                                current: Number(bootstrapEvent.progress),
+                                total: 100,
+                            },
+                        });
+                    }
+                },
+            );
+            try {
+                await tor.start();
+                tor.torController.removeAllListeners();
+            } catch (error) {
```

Comment:
maybe logger could be useful also in catch block

Reply:
If there is an error here it will be logged higher in `packages/suite-desktop/src-electron/modules/index.ts` in `initTorModule`.

We could log it here as well but it will be logged twice then.

Reply:
nooo, ok, all good then